src/client/batch.rs

```diff
@@ -0,0 +1,178 @@
+use std::sync::Arc;
```

Comment:
This file should live in `rskafka::client::producer` since it's producer-specific.

Reply:
786d6e1